isoObject.i

```diff
@@ -37,55 +45,17 @@
 %include "iso22133/iso22133.h"
 %include "udpServer.hpp"
 %include "tcpServer.hpp"
+%include "positioning.h"
 
 
 namespace std {                                                                  
-    %template(TrajectoryWaypointVector) vector<TrajectoryWaypointType>;                                      
+    %template(TrajectoryWaypointVector) vector<TrajectoryWaypointType>;
 }; 
 
 typedef double double_t;
 typedef long int ssize_t;
 
-
 struct timeval {
 long int tv_sec;
 long int tv_usec;
-};
-
-
-typedef struct {
-	double longitudinal_m_s;
-	double lateral_m_s;
-	bool isLongitudinalValid;
-	bool isLateralValid;
-} SpeedType;
-
-typedef struct {
-    double xCoord_m;
-    double yCoord_m;
-    double zCoord_m;
-    double heading_rad;
-    bool isPositionValid;
-    bool isHeadingValid;
-    bool isXcoordValid;
-    bool isYcoordValid;
-    bool isZcoordValid;
-} CartesianPosition;
-
-typedef struct {
-    double latitude_deg;
-    double longitude_deg;
-    double altitude_m;
-    bool isLatitudeValid;
-    bool isLongitudeValid;
-    bool isAltitudeValid;
-} GeographicPositionType;
-
-
-
-typedef struct {
-	double longitudinal_m_s2;
-	double lateral_m_s2;
-	bool isLongitudinalValid;
-	bool isLateralValid;
-} AccelerationType;
```

Comment:
I think you need these to be able to initialize a corresponding type in the target language